scoring/tests/test_end_state_scoring.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+import unittest
+
+import yaml
+
+# Path hackery
+import pathlib
+import sys
+ROOT = pathlib.Path(__file__).parent.parent
+sys.path.insert(0, str(ROOT))
+
+from score import EndStateScorer as Scorer
+
+class ScorerTests(unittest.TestCase):
```

Comment:
Would be good to have some tests here for the case where a territory becomes unclaimed after having been claimed. (Maybe even add them to the other scorer's tests too if it doesn't have them)

Reply:
https://github.com/srobo/sr2021-comp/pull/4/commits/831f04b295535e382976de946d2a3b65b835ecbb